src/main/java/org/spongepowered/common/mixin/core/world/MixinWorld.java

```diff
@@ -355,6 +356,11 @@ public boolean setBlock(int x, int y, int z, BlockState block, Cause cause) {
 
 
     @Override
+    public Location<World> getHighestBlockAt(Vector3i position) {
+        return new Location<World>(this, position.getX(), this.func_189649_b(position.getX(), position.getZ()), position.getZ());
```

Comment:
This is going to return `(x, 0, z)` when called in an unloaded chunk, I don't feel like this is the best behaviour for us to return but I'm not sure what the best thing for us to do is.